go/src/socialapi/workers/oauth/api/api.go

```diff
@@ -1,100 +0,0 @@
-package main
```

Comment:
I did not find any reference to it, thus I assumed it's not being used.

Moreover it does not build - so it could not be used in current form:

```
$ pwd
/Volumes/rjk.io/src/github.com/koding/koding/go/src/socialapi/workers/oauth/api
$ go build .
can't load package: package socialapi/workers/oauth/api: found packages main (api.go) and api (handlers.go) in /Volumes/rjk.io/src/github.com/koding/koding/go/src/socialapi/workers/oauth/api
```